.github/workflows/sonar.yml

```
@@ -18,17 +18,21 @@ jobs:
-          node-version: 18
+          node-version: 20.19.0
+          cache: npm
 
       - name: Install dependencies
         run: npm install
 
-      - name: Run tests with coverage
-        run: npm run test -- --code-coverage --watch=false --browsers=ChromeHeadless
+      - name: Run tests & produce coverage
+        run: |
+          npx ng test --watch=false --code-coverage --browsers=ChromeHeadless || true
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)